### ISSUE TITLE
Unable to create new template Error while querying work items: VS403474

### ISSUE THREAD

**Issue by AndrewJNorris** [microsoft/AzureDevOpsDemoGenerator#386]
I'm trying to create a custom template but get the following error:
"Error while querying work items: VS403474: You requested 212 work items which exceeds the limit of 200"
This project has only 92 work items so it isn't hitting the 200 work items limit.

![image](https://user-images.githubusercontent.com/51118973/108705693-8f09c300-7505-11eb-899f-13d8db0b2c8c.png)

[AB#3602](https://vstsdemodata.visualstudio.com/89b9b739-9a1c-4739-8aae-6a8a1eb67ee7/_workitems/edit/3602)

**Comment by AndrewJNorris** [comment]
This failed for me several times on Friday 19th Feb and again this morning.
I then created a new project, made no changes to it and imported that to the generator and it worked.

Since then I have been able to import the original, problem project project to the generator and download the resultant zip file.

I can also create a new project from that zip file template and that gives no errors. However, it has no work items in it at all.
